port/common/omrport.tdf

```diff
@@ -1630,3 +1630,14 @@ TraceEvent=Trc_PRT_sl_open_shared_library_noload Group=sl Overhead=1 Level=3 NoE
 TraceException=Trc_PRT_retrieveLinuxMemoryStats_failedOpeningSwappinessFs Group=sysinfo Overhead=1 Level=1 NoEnv Template="retrieveLinuxMemoryStats: Failed to open /proc/sys/vm/swappiness. Error code = %d."
 TraceException=Trc_PRT_retrieveLinuxMemoryStats_failedReadingSwappiness Group=sysinfo Overhead=1 Level=1 NoEnv Template="retrieveLinuxMemoryStats: Failed to read /proc/sys/vm/swappiness. Error code = %d."
 TraceException=Trc_PRT_retrieveLinuxMemoryStats_unexpectedSwappinessFormat Group=sysinfo Overhead=1 Level=1 NoEnv Template="retrieveLinuxMemoryStats: Expected %d items to read, but read %d items."
+
+TraceEntry=Trc_PRT_sysinfo_get_process_start_time_enter Group=sysinfo Overhead=1 Level=1 NoEnv Template="Enter omrsysinfo_get_process_start_time for pid %llu"
+TraceEntry=Trc_PRT_sysinfo_get_process_start_time_stat_error Group=sysinfo Overhead=1 Level=1 NoEnv Template="stat error for pid %llu"
+TraceEntry=Trc_PRT_sysinfo_get_process_start_time_sysctl_error Group=sysinfo Overhead=1 Level=1 NoEnv Template="sysctl error for pid %llu"
+TraceEntry=Trc_PRT_sysinfo_get_process_start_time_pid_does_not_exist Group=sysinfo Overhead=1 Level=1 NoEnv Template="pid %llu does not exist"
+TraceEntry=Trc_PRT_sysinfo_get_process_start_time_getprocs_error Group=sysinfo Overhead=1 Level=1 NoEnv Template="getprocs error for pid %llu"
+TraceEntry=Trc_PRT_sysinfo_get_process_start_time_BPX4GTH_error Group=sysinfo Overhead=1 Level=1 NoEnv Template="BPX4GTH error for pid %llu"
+TraceEntry=Trc_PRT_sysinfo_get_process_start_time_BPX1GTH_error Group=sysinfo Overhead=1 Level=1 NoEnv Template="BPX1GTH error for pid %llu"
+TraceEntry=Trc_PRT_sysinfo_get_process_start_time_OpenProcess_error Group=sysinfo Overhead=1 Level=1 NoEnv Template="OpenProcess error for pid %llu"
+TraceEntry=Trc_PRT_sysinfo_get_process_start_time_GetProcessTimes_error Group=sysinfo Overhead=1 Level=1 NoEnv Template="GetProcessTimes error for pid %llu"
```

Comment:
Include the function name in the tracepoint message to highlight the source of the tracepoint.